drivers/hmis/spec/requests/hmis/update_referral_posting_spec.rb

```diff
@@ -56,4 +56,34 @@
       expect(errors).to be_empty
     end.to change(Hmis::Hud::Enrollment.where(id: e1.id).exited, :count).by(1)
   end
+
+  context 'when an enrollment already exists' do
+    let!(:coc1) { create :hud_project_coc, data_source: ds1, project_id: p1.project_id, state: 'MA' }
+    let!(:access_control) { create_access_control(hmis_user, p1) }
+
+    let!(:c2) { create(:hmis_hud_client, data_source: ds1) }
+    let!(:r2) { create :hmis_external_api_ac_hmis_referral }
+    let!(:r2_hhm) { create :hmis_external_api_ac_hmis_referral_household_member, client: c2, referral: r2 }
+    let!(:rp2) { create :hmis_external_api_ac_hmis_referral_posting, identifier: nil, referral: r2, project: p1, status: 'assigned_status', unit_type: nil }
+
+    it 'should error on both WIP and non-WIP enrollment' do
```

Comment:
can we add a test case to ensure that it doesn't error if a household member has an Exited enrollment in the destination project?